mkdocs.yml

```diff
@@ -1,13 +1,17 @@
 site_name: "Pi-hole documentation"
 site_url: https://docs.pi-hole.net
-repo_url: "https://github.com/pi-hole/pi-hole/"
+repo_url: "https://github.com/pi-hole/pi-hole"
 edit_uri: ""
-copyright: 'Copyright &copy; 2018 Pi-hole LLC'
+copyright: 'Copyright &copy; 2019 Pi-hole LLC'
```

Comment:
And it will be 2020 before we even know it...!

I wonder if there is a way we can pull in a `${currentYear}` type value here. One less thing to remember every Jan 1st...

Reply:
I'm not an expert with mkdocs but from a quick look I don't think it's possible without further modifications to the template files

https://github.com/squidfunk/mkdocs-material/blob/210139c0d3e8f1ce4aa9872c39dcbdbfb24ec36b/src/partials/footer.html#L86-L90

Reply:
Yeah, more just thinking out loud really. Maybe I will just make a bot that fires on 1st Jan every year and submits a PR to change the copyright year :D

Reply:
I think it's doable, we just need to request it upstream or we just overwrite the template partial :)

https://github.com/zendframework/zf-mkdoc-theme/commit/597f50335f80a34cfa3270c2fada9a64bbcbc896

Reply:
Doesn't matter. Copyrights start from the date of creation and don't need to be updated for coverage. 

https://stackoverflow.com/a/2391555

Reply:
Fine by me, feel free to revert it. It's just that on the main site it says 2019.

Reply:
I don't think it needs to be reverted, just doesn't need to be watched for automated (or manual) updates at the start of the year.